src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/VectorConfig.java

```diff
@@ -15,27 +16,56 @@
  */
 public record VectorConfig(
     boolean vectorEnabled,
+    String fieldName,
     int vectorSize,
     SimilarityFunction similarityFunction,
     VectorizeConfig vectorizeConfig) {
 
   // TODO: this is an immutable record, this can be singleton
   // TODO: Remove the use of NULL for the objects like vectorizeConfig
   public static VectorConfig notEnabledVectorConfig() {
-    return new VectorConfig(false, -1, null, null);
+    return new VectorConfig(false, null, -1, null, null);
   }
 
-  // convert a vector jsonNode from table comment to vectorConfig
+  // convert a vector jsonNode from table comment to vectorConfig, used for collection
```

Comment:
... not used for tables? So only collection? (if so, comment should state that)

Reply:
I guess it means "used when reading config for Collections (not Tables)"

Reply:
The table there denotes cql table. fwiw I will just say comment to avoid confusion